src/lib/editor/form/EditableSelectField.svelte

```diff
@@ -35,7 +35,7 @@
   on:blur={handleBlur}
   class="{classes} border focus:border-accent-500 custom-select text-base text-ellipsis outline-none w-full rounded h-[32px] pl-3 pr-8 appearance-none cursor-pointer disabled:cursor-default tracking-tight"
 >
-  {#each options as opt (opt.key)}
+  {#each options as opt (opt.value)}
     <option value={opt.value} selected={field.value === opt.value}>{opt.key || opt.value}</option>
```

Comment:
I'd suggest using `opt.label` here if possible for clarity.

Reply:
Done